### PR TITLE
docs: Added the rationale for the working button, changed language to…

### PR DESCRIPTION
**Pull request by ActuallyACat**
# Objective
Added the rationale for the working button, and changed the language to be more prescriptive.

# Motivation and Context 
UX feedback we've had from the Working button means it's possible for sibling controls (such as buttons) to change position when a button moves to/from `working` state. The guideline recommends that a working button retain it's pre-working state width. 

I've also changed the language around working button accessibility to be more prescriptive and added an example for additional context (thanks @cmrn !) 

cc/ @dougmacknz 

workingbutton

# Checklist

[ x ] If this contains visual changes, has it been reviewed by a designer? 
[ x ] I have considered likely risks of these changes and got someone else to QA as appropriate
[ x ] I have or will communicate these changes to the front end practice

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask Design Systems team to review it to catch any issues.

### CODE REVIEW

**Comment by ActuallyACat** [comment]
@katsambarcus to follow up with a figma embad
@ActuallyACat to check/understand our API timeouts

**Comment by katsambarcus** [comment]
Figma embed which includes a visual example to be included near the Working state guidelines

<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D84287%253A39155" allowfullscreen></iframe>

File: site/docs/guidelines/interaction-states.mdx [hunk 1]
```diff
@@ -49,22 +49,23 @@ A disabled state applies to an element when the action is unavailable to a user.
   - If you need to disable a button, help users (including people using assistive technologies, such as screen reader) understand how to enable it again. For example, include nearby microcopy that explains why it is disabled and what action the user can take e.g. “There are no more reports to share. To share more reports, wait for a survey to close.”
 
 
-### Read-only
+### Read-only State
```
Comment (dlimiter) [inline site/docs/guidelines/interaction-states.mdx:53]: Our capitalisation rules are a bit of a mess, but I think "State" should be lower case.

File: site/docs/guidelines/interaction-states.mdx [hunk 2]
```diff
@@ -49,22 +49,23 @@ A disabled state applies to an element when the action is unavailable to a user.
...
 
 A "read-only" form element cannot be modified, but can otherwise be focused, read, and submitted with a form. For example, you might show an employee what other previously known information about them by their organization that they cannot edit within Culture Amp would be submitted along with their user-provided data in a form. This state is rarely needed. More often, it would be provided as regular text content instead of a read-only form element.
 
+### Working State
```
Comment (dlimiter) [inline site/docs/guidelines/interaction-states.mdx:56]: Lower case "state"

**Review by dlimiter — APPROVED** [review]

**Comment by ActuallyACat** [comment]
Fixed - thanks @dlimiter !

**Pull request merged**